lib/handlers.py

```diff
@@ -278,18 +289,54 @@ def _run_auto_complete(view):
             'next_completion_if_showing': False,
         })
 
+    def _flatten_completions(self, completions, nesting=0):
+        if not completions:
+            return []
+
+        result = []
+        for c in completions:
+            result.append((self._brand_completion('  ' * nesting + c['display'], c['hint']),
```

Comment:
We should remove the indentation from the nesting for Sublime.

Reply:
I've removed the indentation and added the new settings.
@its-dhung